docs/Tutorials/zot-registry-tutorial.mdx

```diff
@@ -0,0 +1,236 @@
+```
+
+**Step 2: Authenticate with zot registry**
+
+It is to be noted that in case of zot, authentication is disabled by default. Still, if you wish to authenticate, you can either use `docker login` or [`oras login`](https://oras.land/docs/CLI_Reference/oras_login) to do so.
+
+**Step 3: Pushing artifacts to the zot repository**
+
+```
+oras push --plain-http 192.168.49.2:30276/hello-artifact:v1 \
+        --artifact-type application/vnd.acme.rocket.config \
+        artifact.txt:text/plain -d -v
+```
+
+In this command, we have used the output we got in step 5 of phase 1.
+
+```
+Uploading a948904f2f0f artifact.txt
+Uploaded  a948904f2f0f artifact.txt
+Pushed [registry] 192.168.49.2:30276/hello-artifact:v1
+Digest: sha256:bcdd6799fed0fca0eaedfc1c642f3d1dd7b8e78b43986a89935d6fe217a09cee
+```
+
+After pushing the artifact, we can see them on the same link:
+
+![](https://hackmd.io/_uploads/SJdQHUKSn.png)
+
+
+**Step 4: Pulling artifacts from the zot repository**
+
+```
+oras pull --plain-http 192.168.49.2:30276/hello-artifact:v1 -d -v
+```
+
+```
+DEBU[0000] > Request URL: "http://192.168.49.2:30276/v2/hello-artifact/manifests/v1" 
+DEBU[0000] > Request method: "GET"                      
+DEBU[0000] > Request headers: 
+
+<!-- truncate -->
+
+Downloaded  a948904f2f0f artifact.txt
+Downloaded  bcdd6799fed0 application/vnd.oci.image.manifest.v1+json
+Pulled [registry] 192.168.49.2:30276/hello-artifact:v1
+Digest: sha256:bcdd6799fed0fca0eaedfc1c642f3d1dd7b8e78b43986a89935d6fe217a09cee
+```
+
+**Step 5: To attach files to an existing artifact**
+
+First, let's create another sample file to add,
+
+```
+echo "hello world" > hi.txt
+```
+
+You can use the command below to attach `hi.txt` to the artifact we pushed above:
+
+```
+oras attach --plain-http --artifact-type doc/example 192.168.49.2:30276/hello-artifact:v1 hi.txt
+```
+
+```
+Exists    a948904f2f0f hi.txt
+Attached to [registry] 192.168.49.2:30276/hello-artifact@sha256:327db68f73d0ed53d528d927a6703c00739d7c1076e50762c3f6641b51b76fdc
+Digest: sha256:bcdd6799fed0fca0eaedfc1c642f3d1dd7b8e78b43986a89935d6fe217a09cee
+```
+
+**Step 6: To check the referrers of the manifest in our zot registry**
+
+```
+oras discover --plain-http 192.168.49.2:30276/hello-artifact:v1
+```
+
+```
+Discovered 1 artifact referencing v1
+Digest: sha256:327db68f73d0ed53d528d927a6703c00739d7c1076e50762c3f6641b51b76fdc
+
+Artifact Type   Digest
+doc/example     sha256:bcdd6799fed0fca0eaedfc1c642f3d1dd7b8e78b43986a89935d6fe217a09cee
+```
+
+**Note:** We have used the flag `--plain-http` in the steps because we want to allow insecure connections to registry without SSL check.
+
+### Conclusion
+
+You can now successfully push OCI artifacts to your zot registry! 
+
+Distributing OCI artifacts to OCI registries aids in collaboration and code sharing. It enables teams to acquire and use standardized container images. This is why it is considered to play a crucial role in maintaining consistency among teams.
```

Comment:
```suggestion
Distributing OCI artifacts to OCI registries aids in collaboration and code sharing. It enables teams to acquire and use through a standardized artifact interface. This is why it is considered to play a crucial role in maintaining consistency among teams.
```
I'm not entirely happy with that suggestion, but maybe something along those lines

Reply:
Okay, I'll change it to that for now and try to come up with something along those lines